File: Auction_Project/Models/Bids/BidResponseDTO.cs
using Auction_Project.Models.Items;
using Auction_Project.Models.Users;

namespace Auction_Project.Models.Bids
{
    public class BidResponseDTO
    {
        public ItemResponseDTO ItemResponse { get; set; } 
        public UserResponseDTO UserResponse { get; set; }

        public decimal BidPrice { get; set; }
    }
}

File: Frontend/src/app/api/models/bid-response-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ItemResponseDto } from './item-response-dto';
import { UserResponseDto } from './user-response-dto';
export interface BidResponseDto {
  bidPrice?: number;
  itemResponse?: ItemResponseDto;
  userResponse?: UserResponseDto;
}
